oozie-to-airflow/init.sh

```diff
@@ -17,7 +17,7 @@
 command -v pip >/dev/null 2>&1 || { echo >&2 "pip appears to not be installed. Aborting."; exit 1; }
 
 # Check for python3
-command -v python3 >/dev/null 2>&1 || { echo >&2 "python 3 does not appear to be installed. Aborting."; exit 1;}
+command -v python3 >/dev/null 2>&1 || { echo >&2 "python 3 does not appear to be installed. Aborting."; exit 1; }
```

Comment:
I haven't used git in a while so feel free to correct me.

Is this a result of merging in changes from upstream into your working branch? If so, would rebasing help prevent his?